File: GestaoApi/Models/Pessoa.cs
using System.ComponentModel.DataAnnotations.Schema;
using System.ComponentModel.DataAnnotations;

namespace GestaoApi.Models
{
    [Table("Pessoa")]
    public partial class Pessoa
    {
        [Key]
        [DatabaseGenerated(DatabaseGeneratedOption.Identity)]
        public long Id { get; set; }

        public string? Nome { get; set; }

        public string? Sobrenome { get; set; }

        public ICollection<PessoaXCargo> IdPessoaXCargos { get; set; } = new List<PessoaXCargo>();

        public PessoaXRecursoEmpresa? IdPessoaXRecursoEmpresa { get; set; }

    }
}

File: GestaoAngular/src/app/Pessoa.ts
import { Cargo } from "./Cargo";

export class Pessoa {
    id!: number;
    nome!: string;
    sobrenome!: string;
    cargos?: Cargo[];
}

export interface PessoaXCargo {
    idPessoa: number;
    idCargo: number;
}
